spec/game_spec.rb

```diff
@@ -32,78 +32,91 @@
-    end
+      it 'can return an array of all game objects' do
+        expect(Game.all).to all be_a Game
+        expect(Game.all.count).to eq 31
+      end
 
-    it '#percentage_home_wins returns correct return value' do
-      expect(Game.percentage_home_wins).to be_a Float
-      expect(Game.percentage_home_wins).to eq 0.48
-    end
+      it '#percentage_home_wins returns correct return value' do
+        expect(Game.percentage_home_wins).to be_a Float
+        expect(Game.percentage_home_wins).to eq 0.48
+      end
 
-    it '#percentage_visitor_wins returns correct return value' do
-      expect(Game.percentage_visitor_wins).to be_a Float
-      expect(Game.percentage_visitor_wins).to eq 0.29
-    end
+      it '#percentage_visitor_wins returns correct return value' do
+        expect(Game.percentage_visitor_wins).to be_a Float
+        expect(Game.percentage_visitor_wins).to eq 0.29
+      end
 
-    it '#percentage_ties returns correct return value' do
-      expect(Game.percentage_ties).to be_a Float
-      expect(Game.percentage_ties).to eq 0.23
-    end
+      it '#percentage_ties returns correct return value' do
+        expect(Game.percentage_ties).to be_a Float
+        expect(Game.percentage_ties).to eq 0.23
+      end
 
-    it 'can return a count of games by season' do
-      expected = {
-        20122013 => 9,
-        20132014 => 4,
-        20142015 => 9,
-        20152016 => 2,
-        20162017 => 3,
-        20172018 => 4
-      }
-      expect(Game.count_of_games_by_season).to eq (expected)
-    end
+      it 'can return a count of games by season' do
+        expected = {
+          20122013 => 9,
+          20132014 => 4,
+          20142015 => 9,
+          20152016 => 2,
+          20162017 => 3,
+          20172018 => 4
+        }
+        expect(Game.count_of_games_by_season).to eq (expected)
+      end
 
-    it 'can return the total number of goals in a season' do
-      expected = {
-        20122013 => 43,
-        20132014 => 13,
-        20142015 => 39,
-        20152016 => 7,
-        20162017 => 15,
-        20172018 => 17
-      }
-      expect(Game.count_of_goals_by_season).to eq (expected)
-    end
+      it 'can return the total number of goals in a season' do
+        expected = {
+          20122013 => 43,
+          20132014 => 13,
+          20142015 => 39,
+          20152016 => 7,
+          20162017 => 15,
+          20172018 => 17
+        }
+        expect(Game.count_of_goals_by_season).to eq (expected)
+      end
+
+      it 'can return the average number of goals by season' do
+        expected = {
+          20122013 => 4.78,
+          20132014 => 3.25,
+          20142015 => 4.33,
+          20152016 => 3.50,
+          20162017 => 5.00,
+          20172018 => 4.25
+        }
+
+        expect(Game.average_goals_by_season).to eq (expected)
+      end
+
+      it "gets the highest total score from all games" do
+        expect(Game.highest_total_score).to eq(11)
```

Comment:
This test is currently failing. The Spec Harness passes, which tells me the method is fine, so I'm guessing this is just because these tests pull from the dummy files rather than the data file.  Change this to 9 and it should pass.